src/components/slotMapper/SlotMapper.tsx

```diff
@@ -263,6 +263,9 @@ function SlotMapper({
             handleCopySlots(outputAddress);
             return;
           }
+        } else {
```

Comment:
Not sure I understand why this was added.

Reply:
Sorry, this is the fix for a bug that I happened to see while testing in Visium cDNA related to transferring samples from slide to plate (x792)